Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java

```diff
@@ -137,54 +139,56 @@ private static String getEmailAddressAttrDisplayName() {
         List<CorrelationAttributeInstance> correlationAttrs = new ArrayList<>();
-        try {
-            int artifactTypeID = analysisResult.getArtifactTypeID();
-            if (artifactTypeID == ARTIFACT_TYPE.TSK_INTERESTING_ARTIFACT_HIT.getTypeID()) {
-                BlackboardAttribute assocArtifactAttr = analysisResult.getAttribute(BlackboardAttribute.Type.TSK_ASSOCIATED_ARTIFACT);
-                if (assocArtifactAttr != null) {
-                    BlackboardArtifact sourceArtifact = Case.getCurrentCaseThrows().getSleuthkitCase().getBlackboardArtifact(assocArtifactAttr.getValueLong());
-                    if (sourceArtifact instanceof DataArtifact) {
-                        correlationAttrs.addAll((CorrelationAttributeUtil.makeCorrAttrsForSearch((DataArtifact) sourceArtifact)));
-                    } else if (sourceArtifact instanceof AnalysisResult) {
-                        correlationAttrs.addAll((CorrelationAttributeUtil.makeCorrAttrsForSearch((AnalysisResult) sourceArtifact)));
-                    } else {
-                        String sourceName = sourceArtifact != null ? "SourceArtifact display name: " + sourceArtifact.getDisplayName() : "SourceArtifact was null";
-                        logger.log(Level.WARNING, "Source artifact found through TSK_ASSOCIATED_ARTIFACT attribute was not a DataArtifact or "
-                                + "an Analysis Result. AssociateArtifactAttr Value: {0} {1}",
-                                new Object[]{assocArtifactAttr.getValueString(), sourceName});
-                    }
+        if (CentralRepository.isEnabled()) {
+            try {
+                int artifactTypeID = analysisResult.getArtifactTypeID();
+                if (artifactTypeID == ARTIFACT_TYPE.TSK_INTERESTING_ARTIFACT_HIT.getTypeID()) {
+                    BlackboardAttribute assocArtifactAttr = analysisResult.getAttribute(BlackboardAttribute.Type.TSK_ASSOCIATED_ARTIFACT);
+                    if (assocArtifactAttr != null) {
+                        BlackboardArtifact sourceArtifact = Case.getCurrentCaseThrows().getSleuthkitCase().getBlackboardArtifact(assocArtifactAttr.getValueLong());
+                        if (sourceArtifact instanceof DataArtifact) {
+                            correlationAttrs.addAll((CorrelationAttributeUtil.makeCorrAttrsForSearch((DataArtifact) sourceArtifact)));
+                        } else if (sourceArtifact instanceof AnalysisResult) {
+                            correlationAttrs.addAll((CorrelationAttributeUtil.makeCorrAttrsForSearch((AnalysisResult) sourceArtifact)));
+                        } else {
+                            String sourceName = sourceArtifact != null ? "SourceArtifact display name: " + sourceArtifact.getDisplayName() : "SourceArtifact was null";
+                            logger.log(Level.WARNING, "Source artifact found through TSK_ASSOCIATED_ARTIFACT attribute was not a DataArtifact or "
```

Comment:
Let's make this message a SEVERE so if it happens in an automated regression test, we will see it.